permissions/resource_permissions_test.go

```diff
@@ -1464,11 +1462,10 @@ func TestPathPermissionsResourceIDFields(t *testing.T) {
 			m = x
 		}
 	}
-	_, err := m.idRetriever(context.Background(), databricks.Must(databricks.NewWorkspaceClient(
-		(*databricks.Config)(config.NewMockConfig(func(r *http.Request) error {
-			return fmt.Errorf("nope")
-		})))), "x")
-	assert.EqualError(t, err, "cannot load path x: nope")
+	w, err := databricks.NewWorkspaceClient(&databricks.Config{})
```

Comment:
What is this change?

Reply:
This happened after upgrading to Go SDK, we no longer have config.NewMockConfig: https://github.com/databricks/databricks-sdk-go/pull/739/files#diff-3434249a481acd0afcf6d4786db5bcae154e942666f2082de7d3f3ff3e44ff01